.circleci/config.yml

```diff
@@ -51,6 +51,7 @@ jobs:
       - run: # Run linter after build because the integration test code depend on compiled typescript...
           name: Linter check
           command: npm run lint:ci
+      - run: npm run typecheck
```

Comment:
As discussed on Slack, let's combine the three `typecheck`s in our `package.json` into one `typecheck` command so we don't need both steps here.